test/helpers/test/shouldFail.test.js

```diff
@@ -92,4 +96,26 @@ describe('shouldFail', function () {
       await assertFailure(shouldFail.outOfGas(this.failer.failWithThrow()));
     });
   });
+
+  describe('reverting.withMessage', function () {
```

Comment:
Can we nest this inside the `reverting` `describe` block?

Reply:
Sure will do